getting_started/setup_vm/ccf-dev.yml

```diff
@@ -12,6 +12,9 @@
     - import_role:
         name: openenclave
         tasks_from: install.yml
+    - import_role:
+        name: llvm_repo
```

Comment:
Should this folder be checked in the PR?

Reply:
Yes! Thank you!